internal/entities/rpc.go

```diff
@@ -18,12 +18,22 @@ const (
 	SuccessStatus  RPCStatus = "SUCCESS"
 )
 
+type RPCEntry struct {
+	Key                   string `json:"key"`
+	XDR                   string `json:"xdr"`
+	LastModifiedLedgerSeq int64  `json:"lastModifiedLedgerSeq"`
+}
```

Comment:
These won't be used anymore, we can clean them up. I had deleted them from the RPC Service PR.

Reply:
Yeap, removed these so you can add them later